EIPS/eip-1654.md

```diff
@@ -0,0 +1,107 @@
+---
+eip: 1654
+title: Off-chain dapp-wallet authentication process with contract wallets support
+author: Maor Zamski (@pazams), Christopher Scott (@chrisaxiom)
+discussions-to: https://github.com/ethereum/EIPs/issues/1654
+status: Draft
+type: Meta
```

Comment:
At best this is `Informational`.  I'm personally pretty against Informational EIPs in general though and think this is just a Good Idea which isn't appropriate for an EIP.